### ISSUE TITLE
didFinishScanning(_ cameraViewController:with scanResults: BRScanResults) delegate method doesn't get called second time

### ISSUE THREAD

**Issue by 51d4** [BlinkReceipt/blinkreceipt-ios#18]
I am using custom camera with `manualCaptureSession` false. With current behaviour, when user attempts to take a photo (taps a button), I implement the process as such:
```
cameraViewController.userSnappedPhoto { (image, isBlurry) in
          cameraViewController.userConfirmedFrame(image)
          cameraViewController.userFinishedScan()
}
```

However, if I decide to repeat this flow second time (user "retaking" pictures without leaving the custom camera view), the `didFinishScanning(_ cameraViewController:with scanResults: BRScanResults)` delegate method is not called after calling `userFinishedScan()`.

Is there an issue with the way this scanning session is being handled? Or can `userFinishedScan()` be called only once during the session? 

Basically, what I want is a "restarting" the scanning session without having to reinitialize the custom camera. And going through the documentation, I did not see methods or notes on how to handle that correctly.

**Comment by dpanzer** [comment]
Hi @51d4 - there is no way to "restart" the scanning session in the sense of clearing all previous results and starting fresh without closing and re-opening the camera controller unfortunately. This is something we could consider although only in the context of a commercial agreement - please reach out to blinkreceipt@microblink.com if you'd like to discuss further.

As the docs specify `didFinishScanning` should be called only "to notify the camera controller that the user has indicated they have finished scanning". So it really does mean they are finished and there's no going back.
